src/game/gameOverModal.scss

```diff
@@ -0,0 +1,47 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.8);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+  &__content {
+    max-width: 90vw;
+    max-height: 90vh;
+
+    h2 {
+      border-bottom: 5px solid white;
+      line-height: 1.4;
+      font-size: 48px;
+      margin: 0;
+      @media screen and(min-width: 767px) {
+        font-size: 144px;
```

Comment:
Move it to a variable. Seriously, is the font size `so big` there? Cannot believe it somehow.

Reply:
I also couldn't believe but yeah. It's 144px at figma project